test/e2e/user_rolebindings_test.go

```diff
@@ -0,0 +1,254 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+	config "sigs.k8s.io/controller-runtime/pkg/client/config"
+
+	"k8s.io/apimachinery/pkg/types"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	userv1 "github.com/openshift/api/user/v1"
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"k8s.io/client-go/rest"
+
+	testsupport "github.com/codeready-toolchain/toolchain-e2e/testsupport"
+)
+
+func TestUserCreatingRoleBindings(t *testing.T) {
+	//os.Setenv("MEMBER_NS", "toolchain-member-24093142")
+	//os.Setenv("MEMBER_NS_2", "toolchain-member2-24093142")
+	//os.Setenv("HOST_NS", "toolchain-host-24093142")
+	//os.Setenv("REGISTRATION_SERVICE_NS", "toolchain-host-24093142")
+	// given
+	awaitilities := testsupport.WaitForDeployments(t)
+
+	//config, err := clientcmd.BuildConfigFromFlags("", "/Users/kanikarana/openshift-install-mac/my_cluster/auth/kubeconfig")
+	//config, err := rest.InClusterConfig()
+	config, err := config.GetConfig()
+	require.NoError(t, err)
+	s := runtime.NewScheme()
+	err = userv1.Install(s)
+	require.NoError(t, err)
+
+	// Create and approve user signup
+	testsupport.NewSignupRequest(t, awaitilities).
+		Username("harleyquinn").
+		ManuallyApprove().
+		TargetCluster(awaitilities.Member1()).
+		EnsureMUR().
+		RequireConditions(testsupport.ConditionSet(testsupport.Default(), testsupport.ApprovedByAdmin())...).
+		Execute()
+
+	//create a non-sandbox user
+	testsupport.NewSignupRequest(t, awaitilities).
+		Username("joker").
+		ManuallyApprove().
+		TargetCluster(awaitilities.Member1()).
+		EnsureMUR().
+		RequireConditions(testsupport.ConditionSet(testsupport.Default(), testsupport.ApprovedByAdmin())...).
+		Execute()
+
+	nonsandboxUser := &userv1.User{}
+	err = awaitilities.Member1().Client.Get(context.TODO(), types.NamespacedName{Name: "joker"}, nonsandboxUser)
+	require.NoError(t, err)
+
+	nonsandboxUser.Labels = map[string]string{
+		toolchainv1alpha1.OwnerLabelKey: "joker",
+	}
+	err = awaitilities.Member1().Client.Update(context.TODO(), nonsandboxUser)
+	require.NoError(t, err)
+	err = awaitilities.Member1().Client.Get(context.TODO(), types.NamespacedName{Name: "joker"}, nonsandboxUser)
+	require.NoError(t, err)
```

Comment:
Yeah, ended up creating just a user (without identity) and a rolebinding to let this user access namespaces of other users. Also, when i was creating a sandbox user and removing its provider label, the provider label kept re-appearing (I did update). I didn't look deep enough into it, but i'm guessing that was because of our controllers (UserAccount watches Users with owner label). 
So there was no other way of creating a non-sandbox user.